mender-convert

```diff
@@ -723,27 +723,41 @@ sudo true
 case "$1" in
   raw-disk-image-shrink-rootfs)
     do_raw_disk_image_shrink_rootfs || rc=$?
-  [[ $rc -ne 0 ]] && { return 1; }
+    [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "Shrinked Linux image you can convert to Mender disk image is at "$raw_disk_image
     ;;
   raw-disk-image-create-partitions)
     do_raw_disk_image_create_partitions || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "The disk image with modified partition table you can find at "$mender_disk_image
     ;;
   install-mender-to-mender-disk-image)
     do_install_mender_to_mender_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "The disk image with configured Mender client you can find at "$mender_disk_image
     ;;
   install-bootloader-to-mender-disk-image)
     do_install_bootloader_to_mender_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "The disk image with installed bootloader for Mender updates you can find at "$mender_disk_image
     ;;
   mender-disk-image-to-artifact)
     do_mender_disk_image_to_artifact || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
+    echo "The Mender Artifact you can upload to your Mender server to deploy to your devices is at "$mender_artifact
     ;;
   from-raw-disk-image)
     do_from_raw_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Conversion complete!"
+    echo "The Mender root file system partition is at "$mender_rootfs_image
+    echo "The Mender image you can provision your device storage with is at "$mender_disk_image
```

Comment:
The Mender disk image you can provision your device storage with is at "$mender_disk_image.

Reply:
Would move this line first, then the Mender root file system partition in the middle (first Mender disk image, then Mender root file system partition, then Mender Artifact).